main.go

```diff
@@ -225,6 +231,41 @@ func main1() int {
 	return 0
 }
 
+func validateOptions() error {
+	var allPassed, nonePassed bool
+	litOptions := strings.Split(flagGarbleLiterals, ",")
+	for i, op := range litOptions {
+		switch op {
+		case "none":
+			if allPassed {
+				return errors.New("error parsing -literals flag: 'all' and 'none' are mutually exclusive")
+			} else if i != 0 {
+				return errors.New("error parsing -literals flag: no other options can be passed with 'none'")
```

Comment:
None should only be implicit (the empty case)

Reply:
Also, I think "none" and "all" should be allowed as the entire string, not as elements. Because what does `all,strings` really mean?